frontend/src/app/reducers/browser.js

```diff
@@ -8,7 +8,11 @@ const isUserAgentMobile = isMobile(userAgent);
 const isUserAgentMinFirefox = isMinFirefoxVersion(userAgent, config.minFirefoxVersion);
 const locale = (navigator.language || '').split('-')[0];
 
-export default handleActions({}, {
+export const setState = (state) => state;
```

Comment:
Ok, I guess handleActions takes a map keyed with the action type and with reducer values, so I understand this now. I still don't think redux-actions makes for easy to read code.